README.md

```diff
@@ -24,7 +24,7 @@ ref: developer-engagement
 ```
 
 * **layout:** Layout for the page. This has to be ``page``.
-* **title:** (Localizable) Main title for the page.
+* The **title:** (Localizable) Main title for the page.
```

Comment:
I think we should keep this as it was before, as this is solely a list of the possible configuration options.